README.md

```diff
@@ -1,3 +1,5 @@
+![coverage](https://raw.githubusercontent.com/nginx/agent/badges/.badges/v3/coverage.svg)
```

Comment:
This link will only work after a `v3` build has been run after this PR has been merged.